src/DIRAC/Interfaces/API/test/Test_DIRAC.py

```diff
@@ -2,21 +2,20 @@
 """
 # pylint: disable=no-member, protected-access, missing-docstring
 import logging
-
 from pprint import pformat
 import pytest
 from unittest.mock import MagicMock, call
 
-from DIRAC.Interfaces.API.Dirac import Dirac
 from DIRAC import S_OK
+from DIRAC.Interfaces.API.Dirac import Dirac
 
 logging.basicConfig()
 LOG = logging.getLogger("TestDirac")
 LOG.setLevel(logging.ERROR)
 
 
-@pytest.fixture
-def dirac():
+@pytest.fixture(name="dirac")
```

Comment:
Why did you decide to change the names of the fixtures?

Reply:
This change was done because of Pylint

Reply:
This change was done because of Pylint